### PR TITLE
Fix -new inspector- Error when pressing reset button on exported var …

### PR DESCRIPTION
**Pull request by Douglas-W-Williams**
…with no initial default assignment - Fixes one of the issues mentioned in #19534

### CODE REVIEW

File: editor/editor_inspector.cpp [hunk 1]
```diff
@@ -459,7 +459,8 @@ void EditorProperty::update_reload_status() {
 		Ref<Script> scr = object->get_script();
 		Variant orig_value;
 		if (scr->get_property_default_value(property, orig_value)) {
-			if (orig_value != object->get(property)) {
+			Variant o = object->get(property);
```
Comment (akien-mga) [inline editor/editor_inspector.cpp]: Please give the variable a more meaningful name.

Comment (ordigdug) [reply editor/editor_inspector.cpp]: @akien-mga Changed as requested.

**Comment by ordigdug** [comment]
@akien-mga I updated this pull request.

**Comment by akien-mga** [comment]
As discussed in today's PR review meeting, @reduz is strongly against triggering a reload of the script to fix this. The out of sync issue should be fixed at the root instead of corrected further down the road.

The issue described in #19534 will still have to be fixed for 3.1-stable though (might be worth splitting the remaining valid issues from #19534 in their own ticket).

**Pull request closed without merging**